examples/volumes/glusterfs/README.md

```diff
@@ -1,30 +1,39 @@
 ## GlusterFS
 
-[GlusterFS](http://www.gluster.org) is an open source scale-out filesystem. These examples provide information about how to allow containers use GlusterFS volumes.
+[GlusterFS](http://www.gluster.org) is an open source scale-out filesystem.
+These examples provide information about how to allow containers use GlusterFS
+volumes.
 
-There are couple of ways to use GlusterFS as a persistent data store in application pods.
+There are couple of ways to use GlusterFS as a persistent data store in
+application pods.
 
-*) Static Provisioning of GlusterFS Volumes.
-*) Dynamic Provisioning of GlusterFS Volumes.
+* Static Provisioning of GlusterFS Volumes.
+* Dynamic Provisioning of GlusterFS Volumes.
 
 ### Static Provisioning
 
-Static Provisioning of GlusterFS Volumes is analogues to creation of a PV ( Persistent Volume) resource by specifying the parameters in it. This
-also need a working GlusterFS cluster/trusted pool available to carve out GlusterFS volumes.
+Static Provisioning of GlusterFS Volumes is analogues to creation of a PV (
```

Comment:
Ah, I normally try and fix those up after I re-wrap stuff, but I guess I missed it because of the space after the paren. I'll push up a new change here in a second.